tests/integration/routes/test_thank_you.py

```diff
@@ -103,3 +103,13 @@ def test_back_to_surveys_link_on_thank_you(self):
         self.assertInUrl("thank-you")
         self.assertInBody("Back to surveys")
         self.assertInBody(ACCOUNT_SERVICE_TODO_PATH)
+
+    def test_answer_availability_text(self):
```

Comment:
I think something like `test_view_answers_after_submission_guidance` would be clearer.